app/src/main/java/io/intrepid/contest/screens/adminstatus/AdminStatusPresenter.java

```diff
@@ -12,7 +12,7 @@
 import io.reactivex.disposables.Disposable;
 import timber.log.Timber;
 
-class AdminStatusPresenter extends BasePresenter<AdminStatusContract.View> implements AdminStatusContract.Presenter, ConfirmStartContestDialog.DialogInteractionListener {
+public class AdminStatusPresenter extends BasePresenter<AdminStatusContract.View> implements AdminStatusContract.Presenter, ConfirmStartContestDialog.DialogInteractionListener {
```

Comment:
I get the following error when they're package local: `'public' subclass exposes its 'public/*package*/' supertype argument ResultsAvailablePresenter`

Apparently this is one area of discussion:
https://discuss.kotlinlang.org/t/kotlin-to-support-package-protected-visibility/1544/20
http://stackoverflow.com/questions/35460253/why-public-class-cant-extend-private-class

Maybe once the presenters are in kotlin it can be changed to something other than `public`? I haven't found a way around it...